README.md

```diff
@@ -58,6 +58,32 @@ Before opening a PR, check the service renders successfully at http://localhost:
 and run all the tests locally as described below.
 
 ## Adding to your service
+
+### Users of the latest versions of play-ui or play-frontend-hmrc
+
+If you are using the latest version of [hmrc/play-ui](https://github.com/hmrc/play-ui#accessibility-statements)
+ or [hmrc/play-frontend-hmrc](https://github.com/hmrc/play-frontend-hmrc#accessibility-statements), you can add the 
+ `accessibility-statement.service-path` key to your application.conf. This key is 
+ the path to your accessibility statement under https://www.tax.service.gov.uk/accessibility-statement.
+                                                                       
+For example, if your accessibility statement is https://www.tax.service.gov.uk/accessibility-statement/discounted-icecreams, 
+this property must be set to `/discounted-icecreams` as follows:
+
+```
+accessibility-statement.service-path = "/discounted-icecreams"
+```
+
+Once this is set, the play-ui [FooterLinks](https://github.com/hmrc/play-ui/blob/master/src/main/twirl/uk/gov/hmrc/play/views/layouts/FooterLinks.scala.html)
+  component will auto-generate the correct link to your accessibility statement, including
+the full referrerUrl parameter as described below. Likewise, the new
+ [hmrcFooter](https://github.com/hmrc/play-frontend-hmrc/blob/master/src/main/play-26/twirl/uk/gov/hmrc/hmrcfrontend/views/components/hmrcFooter.scala.html)
+ component will deliver the full govukFooter including the standardised links.
+ 
+Also available is the [hmrcFooterItems](https://github.com/hmrc/play-frontend-hmrc/blob/master/src/main/scala/uk/gov/hmrc/hmrcfrontend/views/config/HmrcFooterItems.scala) helper
+for occasions where it is not convenient to use hmrcFooter.
+
+### Users of older versions of play-ui or Java-based services
+ 
 When adding to your service, an additional parameter should be added to your query string, 
 to help end users report any accessibility that they find. this is:
```

Comment:
Should be upper case "t" in "this is"

Reply:
Just realised that's not part of your changes, but if you could do anyway, would be great!